active/NavigationView/HierarchicalNav/HierarchicalNavigationView.md

```diff
@@ -0,0 +1,265 @@
+There's more detail about the [ItemInvoked](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.iteminvoked?view=winui-2.3) and [SelectionChanged](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.selectionchanged?view=winui-2.3) events later in this document.
+When providing users with a hierarchical tree of navigation options, you may choose to make parent items non-selectable, for example when your app doesn't have a destination page associated with that parent item.
+To prevent an item from showing the selection indicator when invoked, set its [SelectsOnInvoked](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationviewitem.selectsoninvoked?view=winui-2.3) property to False.
+
+Selected items will draw their selection indicators along their left edge when in left mode or their bottom edge when in top mode. 
+The selected item may not always remain visible.
+For example, the selected item may be a child node inside a non-expanded subtree.
+In this situation, the first visible ancestor of the selected item will show as selected, and the selection indicator will move as users expand the subtree. 
+The entire navigation view will show no more than one selection indicator.
+
+## Keyboarding
+Users can move focus around the navigation view using their [keyboard](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions). 
+The arrow keys expose "inner navigation" within the pane and follow the interactions provided in [tree view](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tree-view).
+- **Up arrow** moves focus to the item directly above the item currently in focus
+- **Down arrow** moves focus the item directly below the item currently in focus; note that the items do not need to be visually adjecent, focus will move from the last item in the pane's list to the settings item.
+- **Right arrow**
+  - if the current item is a collapsed parent with children, right arrow expands the item but does not move focus
+  - if the current item is an expanded parent with children, right arrow moves focus to the first child
+  - if the current item is a leaf node, right arrow does nothing
+- **Left arrow** 
+  - if the current item is a collapsed parent with children, left arrow does nothing
+  - if the current item is an expanded parent with children, left arrow collapse the item but does not move focus
+  - if the current item is a leaf node, left arrow moves focus to the current item's parent
+
+# Examples
+
+## Add items in markup
+Declare app navigation hierarchy in markup.
+
+```Xaml
+<muxc:NavigationView>
+    <muxc:NavigationView.MenuItems>
+        <muxc:NavigationViewItem Content="Home" Icon="Home" ToolTipService.ToolTip="Home"/>
+        <muxc:NavigationViewItem Content="Collections" Icon="Keyboard" ToolTipService.ToolTip="Collections">
+            <muxc:NavigationViewItem.MenuItems>
```

Comment:
>MenuItems [](start = 37, length = 9)

Can we make MenuItems the content property of NavigationViewItem ? That way you do not require this line in  markup.

Reply:
oops...this is left over from early days, I need to update

Reply:
This might be a bit confusing since a NavigationViewItem has a content presenter as well.

Reply:
Resolving this conversation because I believe the sample code is correct and we are not able to follow the suggestion.